src/components/DAO/DAO.tsx

```diff
@@ -1,22 +1,540 @@
+import { useEffect, useState } from "react";
+import { useStore } from "@nanostores/react";
+import { formatUnits } from "viem";
+import { vaults, publicClient, balances } from "@store";
+import { PlatformABI, platform, ERC20ABI, IERC721Enumerable } from "@web3";
+import type { PlatformData, GitHubUser, ProfitTokenData } from "@types";
+import tokenlist from "../../stability.tokenlist.json";
+
 function DAO() {
+  const [_platformData, setPlatformData] = useState<PlatformData | undefined>(
+    undefined
+  );
+  const [profitTokenData, setProfitTokenData] = useState<ProfitTokenData>();
+  const [prTotalSupply, setPmTotalSupply] = useState("");
+  const [sdivTotalSupply, setSdivTotalSupply] = useState("");
+  const [members, setMembers] = useState<GitHubUser[]>([]);
+
+  const $publicClient = useStore(publicClient);
+  const $vaults = useStore(vaults);
+  const $balances = useStore(balances);
+
+  useEffect(() => {
+    getTeamData();
+    fetchPlatformData();
+  }, [$balances]);
+
+  const getTeamData = async () => {
+    try {
+      const membersData = await fetch(
+        "https://api.github.com/orgs/stabilitydao/public_members"
+      );
+      const dataJson = await membersData.json();
+      setMembers(dataJson);
+      console.log(dataJson);
+    } catch (error) {
+      console.log("Fetching getTeamData", error);
+    }
+  };
+
+  const fetchPlatformData = async () => {
+    if ($publicClient && $balances) {
+      try {
+        const platformVersion: any = await $publicClient.readContract({
+          address: platform,
+          abi: PlatformABI,
+          functionName: "platformVersion",
+        });
+
+        const platformFees: any = await $publicClient.readContract({
+          address: platform,
+          abi: PlatformABI,
+          functionName: "getFees",
+        });
+
+        const contractData: any = await $publicClient.readContract({
+          address: platform,
+          abi: PlatformABI,
+          functionName: "getData",
+        });
+
+        const profitTotalSupply = await $publicClient.readContract({
```

Comment:
move addresses in constants like:

const SDIV = "sdiv address"

///// DAO
import {SDIV} from "@constants"

const _sdivTotalSupply = await $publicClient.readContract({
          address: SDIV,
          abi: ERC20ABI,
          functionName: "totalSupply",
        });
        
  for all addresses